src/toolkit/mat_query.h

```diff
@@ -35,6 +35,9 @@ class MatQuery {
   /// Returns the mass fraction of nuclide nuc in the material.
   double mass_frac(std::string nuc);
 
+  /// Returns the mass fraction of nuclide nuc in the material.
```

Comment:
oops!

On Mar 12, 2015, at 9:10 AM, Anthony Scopatz <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/toolkit/mat_query.h<https://github.com/cyclus/cyclus/pull/1093#discussion_r26303611>:

> @@ -35,6 +35,9 @@ class MatQuery {
>    /// Returns the mass fraction of nuclide nuc in the material.
>    double mass_frac(std::string nuc);
>
> +  /// Returns the mass fraction of nuclide nuc in the material.


This docstring needs an update :)

—
Reply to this email directly or view it on GitHub<https://github.com/cyclus/cyclus/pull/1093/files#r26303611>.